admin/services/access-arrangements.service.js

```diff
@@ -19,18 +19,18 @@ accessArrangementsService.getAccessArrangements = async () => {
 /**
  * Submit access arrangements for single pupil
  * @param {Object} submittedData
- * @param {Number} dfeNumber
+ * @param {Number} schoolID
```

Comment:
@danwilliams that should be `schoolId` on all references

Reply:
...it is; sorry, I think I am missing something about the comment?

Reply:
@danwilliams D shouldn't be capital.
`schoolID` -> `schoolId`

Reply:
Well-spotted :)